Program.cs

```diff
@@ -1,7 +1,23 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter a string: ");
+         string iso = Console.ReadLine ();
+         bool isogram = Isogram (iso);
+         Console.WriteLine ($"{iso} is {(Isogram (iso) ? "" : "not ")}an isogram.");

```

Comment:
isogram variable should have been used in here.

Reply:
Removed isogram variable line.